charts/budibase/values.yaml

```diff
@@ -41,14 +36,10 @@ service:
 
 ingress:
   enabled: true
-  aws: false
-  nginx: true
-  certificateArn: ""
   className: ""
   annotations:
-    kubernetes.io/ingress.class: nginx
-    nginx.ingress.kubernetes.io/client-max-body-size: 150M
-    nginx.ingress.kubernetes.io/proxy-body-size: 50m
+    # nginx.ingress.kubernetes.io/client-max-body-size: 150M
```

Comment:
can probably just remove these altogether if there's no NGINX ingress

Reply:
I was in two minds about this. I'm guessing this is here purely for attachment uploading? If so, and Budibase will function just fine without them, I'll remove them.

Reply:
Yep, exactly - and due to the "two NGINXes" you had to reflect that config in both.  Good to remove!